curation/main.py

```diff
@@ -0,0 +1,115 @@
+from fastapi import FastAPI, Query, Request
+from fastapi.staticfiles import StaticFiles
+from typing import Dict
+from curation import version
```

Comment:
This isn't working as intended
```suggestion
from curation.version import __version__
```